packages/client/modules/userDashboard/components/OrgAuthentication/OrgAuthenticationMetadata.tsx

```diff
@@ -102,56 +80,42 @@ const OrgAuthenticationMetadata = (props: Props) => {
     }
     onCompleted()
     commitLocalUpdate(atmosphere, (store) => {
-      store.get(saml!.id)?.setValue(metadata, 'metadata')
+      store.get(saml!.id)?.setValue(metadataURL, 'metadataURL')
     })
     atmosphere.eventEmitter.emit('addSnackbar', {
       message: 'SSO Configured Successfully',
       autoDismiss: 5,
       key: 'submitMetadata'
     })
   }
-  const checkForURL = async (e: React.ClipboardEvent) => {
-    const pastedText = e.clipboardData?.getData('Text')
-    if (!pastedText) return
-    try {
-      const url = new URL(pastedText)
-      const fetchedMetadata = await fetch(url)
-      const fetchedMetadataStr = await fetchedMetadata.text()
-      if (fetchedMetadataStr.startsWith('<?xml')) {
-        setMetadata(fetchedMetadataStr)
-      }
-    } catch {
-      // not a URL
-    }
-  }
   return (
-    <Container>
-      <Section>
-        <div className='flex text-base font-semibold leading-6 text-slate-700'>Metadata</div>
+    <>
+      <div className='px-6 pb-3'>
+        <div className='flex text-base font-semibold leading-6 text-slate-700'>Metadata URL</div>
         <div className={'flex items-center text-sm text-slate-700'}>
-          Paste metadata from your identity provider
+          Paste the metadata URL from your identity provider
         </div>
-      </Section>
-      <InputSection>
-        <BasicTextArea
-          name='metadata'
-          placeholder={`<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" entityID=...`}
-          value={metadata}
-          onChange={(e) => setMetadata(e.target.value)}
-          onPaste={checkForURL}
+      </div>
+      <div className='px-6 pb-3'>
+        <BasicInput
+          name='metadataURL'
+          placeholder={`https://idp.example.com/app/sso/saml/metadata`}
+          value={metadataURL}
+          onChange={(e) => setMetadataURL(e.target.value)}
+          error={undefined}
         />
-      </InputSection>
+      </div>
       <div className={'px-4 text-tomato-500 empty:hidden'}>{error?.message}</div>
```

Comment:
nit: paddings could be slightly tweaked for the error

<img width="327" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/ca98251a-9c62-4536-8448-92a8b969a158">